src/app/page.tsx

```diff
@@ -1,8 +1,21 @@
-import { getAllDatasets, getDataset } from '@/adapters/api'
+import * as api from '@/adapters/api'
+import { loadAuthorizedUser } from './api/auth/utils'
 
 export default function Home() {
+    const user = loadAuthorizedUser()
+    // console.log('user', user)
+
     const run = async () => {
-        const res = await getDataset('Climate Change Impact')
+        const resDataset = await api.getAllDatasets()
+        console.log('all datasets:', resDataset !== undefined)
+        const resSingleDataset = await api.getDataset('Climate Change Impact')
+        console.log('single dataset:', resSingleDataset !== undefined)
+        const resDatasetFromUserEvent = await api.getDatasetFromUserEvent(
+            user?.id as string
+        )
+        console.log('datasetFromUser:', resDatasetFromUserEvent !== undefined)
+        const resEvent = await api.getEvent('wox79ra56il6u88') // swap out for a working id in your pb
+        console.log('events:', resEvent !== undefined)
     }
```

Comment:
I would just remove the logs, but as we are working with mocked data I don't have any comments yet on this part